### PR TITLE
Glasgow | Prati Amalden| Module-Structuring-and-Testing-Data | week 1 | Sprint 1

### PR DESCRIPTION
**Pull request by PratiAmalden**
## Learners, PR Template

Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [x] My changes meet the [requirements](./README.md) of this task

## Changelist

Briefly explain your PR.

## Questions

Ask any questions you have for your reviewer.

### CODE REVIEW

File: Sprint-1/interpret/to-pounds.js [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
+// make sure the string has at least 3 digits by adding 0 at the front if needed
 const pounds = paddedPenceNumberString.substring(
   0,
   paddedPenceNumberString.length - 2
 );
+//extracts the pounds part
 
 const pence = paddedPenceNumberString
   .substring(paddedPenceNumberString.length - 2)
   .padEnd(2, "0");
```
Comment (cjyuan) [inline Sprint-1/interpret/to-pounds.js:20]: Would this code work equally well if we remove `.padEnd(2, "0")`? It seems redundant.

**Review by cjyuan — COMMENTED** [review]
I left my comments in the "Changed Files" view.

File: Sprint-1/exercises/paths.js [hunk 2]
```diff
@@ -15,4 +15,11 @@ const base = filePath.slice(lastSlashIndex + 1);
...
 
 // Create a variable to store the dir part of the filePath variable
+const dir = filePath.slice(o, lastSlashIndex);
+console.log(`The dir part of ${filePath} is ${dir}`);
+
+
 // Create a variable to store the ext part of the variable
+const lastDotIndex = base.lastIndexOf(".");
+const ext = base.slice(lastDotIndex);
+console.log(`The ext part of ${filePath} is ${ext}`);
```
[the remark this answers is not in the document]
Comment (PratiAmalden) [reply Sprint-1/exercises/paths.js:25]: This is what I got
The base part of /Users/mitch/cyf/Module-JS1/week-1/interpret/file.txt is file.txt
The dir part of /Users/mitch/cyf/Module-JS1/week-1/interpret/file.txt is /Users/mitch/cyf/Module-JS1/week-1/interpret
The ext part of /Users/mitch/cyf/Module-JS1/week-1/interpret/file.txt is .txt

File: Sprint-1/explore/chrome.md [hunk 3]
```diff
@@ -11,8 +11,12 @@ In the Chrome console,
...
 What effect does calling the `alert` function have?
+A pop-up alert appear with "Hello world!"
 
 Now try invoking the function `prompt` with a string input of `"What is your name?"` - store the return value of your call to `prompt` in an variable called `myName`.
 
 What effect does calling the `prompt` function have?
+a prompt dialog appear in the browser with "What is your name?"
+
 What is the return value of `prompt`?
+myName
```
[the remark this answers is not in the document]
Comment (PratiAmalden) [reply Sprint-1/explore/chrome.md:22]: - empty string ""
- null

File: Sprint-1/interpret/to-pounds.js (hunk 1 above)
Comment (PratiAmalden) [reply Sprint-1/interpret/to-pounds.js:20]: I deleted it and yes there wasn't any need for it!

File: Sprint-1/exercises/count.js [hunk 4]
```diff
@@ -4,3 +4,5 @@ count = count + 1;
 
 // Line 1 is a variable declaration, creating the count variable with an initial value of 0
 // Describe what line 3 is doing, in particular focus on what = is doing
+
+// adding 1 to the 0
```
[the remark this answers is not in the document]
Comment (PratiAmalden) [reply Sprint-1/exercises/count.js:8]: takes the current value of count and adds 1 to it, then assigns the new value back to count

File: Sprint-1/exercises/decimal.js [hunk 5]
```diff
@@ -3,7 +3,14 @@ const num = 56.5678;
...
+const wholeNumberPart = Math.floor(num);
 // Create a variable called decimalPart and assign to it an expression that evaluates to 0.5678 ( the decimal part of num )
+const decimalPart = num - Math.floor(num);
+
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
+const roundedNum = Math.round(num);
 
 // Log your variables to the console to check your answers
+console.log(wholeNumberPart);
+console.log(decimalPart);
```
[the remark this answers is not in the document]
Comment (PratiAmalden) [reply Sprint-1/exercises/decimal.js:15]: I fixed it.

File: Sprint-1/exercises/initials.js [hunk 6]
```diff
@@ -2,5 +2,8 @@ let firstName = "Creola";
 let middleName = "Katherine";
 let lastName = "Johnson";
 
+let initials = [firstName[0], middleName[0], lastName[0]];
```
[the remark this answers is not in the document]
Comment (PratiAmalden) [reply Sprint-1/exercises/initials.js:5]: Sure! can please check the changes I made?

**Pull request closed without merging**